src/components/BuildPages/BuildPages.tsx

```diff
@@ -127,14 +129,19 @@ export const BuildPages = () => {
 
   const isLogged = !serviceContainerBuild.data?.status || isBuildWithLog(serviceContainerBuild.data.status);
   const isLoggedTooltip = !isLogged ? `Builds with status ${serviceContainerBuild.data!.status} are not logged.` : '';
+  const liveLogLinkTooltip = !isBuilding ? `Not available while not building.` : '';
+  const staticLogLinkTooltip = isBuilding ? `Not available while building.` : '';
 
   const pageTabs = (
     <PageTabs>
       <PageTabsItem url="details">Details</PageTabsItem>
-      <PageTabsItem url="build-log" isDisabled={!isLogged} tooltip={isLoggedTooltip}>
+      <PageTabsItem url="live-log" isDisabled={!isLogged || !isBuilding} tooltip={isLoggedTooltip || liveLogLinkTooltip}>
+        Live Log
+      </PageTabsItem>
+      <PageTabsItem url="build-log" isDisabled={!isLogged || isBuilding} tooltip={isLoggedTooltip || staticLogLinkTooltip}>
```

Comment:
I've notices if there are two disabled tab items next to each other, then they are visually merged. Unfortunately adding `border-right: 2px solid white` does not seem like a good solution because it breaks bottom shadow a little bit. You can try to fix it, or we can fix it in separated PR later.

Reply:
pretty much way PF has designed it
https://www.patternfly.org/components/tabs/#default-tabs

Reply:
Ok, let's keep for now, thanks